### DISCUSSION TITLE
Cannot Install Security Onion 2.3.140 on Esxi 7

### DISCUSSION THREAD

**Discussion by Chizan-Dolrocks** [Security-Onion-Solutions/securityonion#8486] [Unsupported Versions]
Hello,
We tried to install our manager in ESXi but at the end of the installation we encountered this error:

![so-erreur1](https://user-images.githubusercontent.com/84504744/183435028-0cd63ac4-1345-4f3d-a5b0-0e7952054d60.png)

after we visited the file /roo/sosetup.log, we saw that there is a problem with MySQL : 

![so-erreur2](https://user-images.githubusercontent.com/84504744/183435572-a8b4d514-2f05-4dce-a414-f917c1f85d47.png)

How can we solve this problem?
Thanks in advance

**Comment by reyesj2** [comment]
Is this the only error in sosetup.log? Can you provide the full log

**Reply by Chizan-Dolrocks** [reply]
Hello,
I tried to reinstall the security onion, this time the installation is successful but the problem is that we need to disable the internal firewall of the system (firewalld) to access the security onion console. 
I wanted to know why so-allow does not work ? 

Thank you in advance for your response.

**Reply by dougburks** [reply]
If Setup doesn't complete successfully, then it may not update the firewall. Are you able to provide the full sosetup.log as requested previously?

**Reply by Chizan-Dolrocks** [reply]
Here is the full log file /root/sosetup.log:

[logs.txt](https://github.com/Security-Onion-Solutions/securityonion/files/9290831/logs.txt)

Thank you very much for your answer.

**Reply by dougburks** [reply]
Looks like there are multiple failures in the log. I'd suggest trying a fresh installation in a new VM with more storage. 200GB is the bare minimum and you will likely want much more than that for a production deployment:
https://docs.securityonion.net/en/2.3/hardware.html#minimum-specs

**Reply by Chizan-Dolrocks** [reply]
Hello,
we tried to reinstall in a new VM with a 500 GB disk. but i have this problem : 

<img width="328" alt="so-erreur4" src="https://user-images.githubusercontent.com/84504744/184093561-985c73ba-53fd-49f6-bed6-560ad3d29026.png">

Here is the full log file: 

[logs.txt](https://github.com/Security-Onion-Solutions/securityonion/files/9307251/logs.txt)

Thank you for your response.

**Reply by dougburks** [reply]
Looks like there are multiple timeout issues in the log.

Is your ESXi host oversubscribed?

Is it using SSD or slower spinning drives?

Are you able to perform a test installation on a dedicated machine with SSD to see if that works?

**Reply by 004500sk** [reply]
can any one able to tell, how to reach /root/sosetup.log,
/rrot/errors.log file, please

**Reply by dougburks** [reply]
@004500sk Please start a new discussion rather than replying to this old discussion.
